src/SourceBuild/content/prep.sh

```diff
@@ -14,18 +14,19 @@ usage() {
     echo "  --no-bootstrap              Don't replace portable packages in the download source-built artifacts"
     echo "  --no-prebuilts              Exclude the download of the prebuilts archive."
     echo "  --no-sdk                    Exclude the download of the .NET SDK."
-    echo "  --runtime-source-feed       URL of a remote server or a local directory, from which SDKs and"
```

Comment:
@MichaelSimons these two were not used anywhere in the script so I removed them but not sure we want this breaking change for our partners?